tools/traffic/config.go

```diff
@@ -27,6 +28,24 @@ func NewConfig(ctx *cli.Context) (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	securityParams := make([]*core.SecurityParam, len(ctx.GlobalIntSlice(flags.QuorumIDsFlag.Name)))
```

Comment:
Should this wait after this PR https://github.com/Layr-Labs/eigenda/pull/298?

Reply:
that's a good idea. I'll update when #298 gets merged